code/go/0chain.net/blobbercore/allocation/protocol.go

```diff
@@ -52,9 +52,21 @@ func (a *Allocation) LoadTerms(ctx context.Context) (err error) {
 func VerifyAllocationTransaction(ctx context.Context, allocationTx string, readonly bool) (a *Allocation, err error) {
 	var tx = datastore.GetStore().GetTransaction(ctx)
 
+	t, err := transaction.VerifyTransaction(allocationTx, chain.GetServerChain())
```

Comment:
we need to query transaction API instead of verifying transaction by tx, use allocation ID and GET /v1/screst/6dba10422e368813802877a85039d3985d96760ed844092319743fb3a76712d7/allocation